wled00/json.cpp

```diff
@@ -37,6 +37,24 @@ void deserializeSegment(JsonObject elem, byte it, byte presetId)
     stop = (len > 0) ? start + len : seg.stop;
   }
 
+  // multiply segment 0 (?) if requested untill all LEDs are used
```

Comment:
I was going to suggest putting this in the UI JavaScript, but having it as a capability of the API itself is a good idea!